doc/source/serve/doc_code/create_deployment.py

```diff
@@ -43,81 +42,59 @@ def __init__(self, path: str) -> None:
-    Deployment.options(name="rep-2", num_replicas=2).deploy("/model/rep-2.pkl")
-
-    # Get the current list of deployments
-    print(serve.list_deployments())
-
-    print("ServerHandle API responses: " + "--" * 5)
-
-    # Method 1) Access each deployment using the ServerHandle API
-    for _ in range(2):
-        for d_name in ["rep-1", "rep-2"]:
-            # Get handle to the each deployment and invoke its method.
-            # Which replica the request is dispatched to is determined
-            # by the Router actor.
-            handle = serve.get_deployment(d_name).get_handle()
-            print(f"handle name : {d_name}")
-            print(f"prediction  : {ray.get(handle.remote(random()))}")
-            print("-" * 2)
-
-    print("HTTP responses: " + "--" * 5)
-
-    # Method 2) Access deployment via HTTP Request
-    for _ in range(2):
-        for d_name in ["rep-1", "rep-2"]:
-            # Send HTTP request along with data payload
-            url = f"http://127.0.0.1:8000/{d_name}"
-            print(f"handle name : {d_name}")
-            print(f"prediction  : {requests.get(url, params= {'data': random()}).text}")
-
-# Output:
-# {'rep-1': Deployment(name=rep-1,version=None,route_prefix=/rep-1),
-# 'rep-2': Deployment(name=rep-2,version=None,route_prefix=/rep-2)}
-#
-# ServerHandle API responses: ----------
-# handle name : rep-1
-# prediction  : (pid: 62636); path: /model/rep-1.pkl; data: 0.600; prediction: 1.292
-# --
-# handle name : rep-2
-# prediction  : (pid: 62635); path: /model/rep-2.pkl; data: 0.075; prediction: 0.075
-# --
-# handle name : rep-1
-# prediction  : (pid: 62634); path: /model/rep-1.pkl; data: 0.186; prediction: 0.186
-# --
-# handle name : rep-2
-# prediction  : (pid: 62637); path: /model/rep-2.pkl; data: 0.751; prediction: 1.444
-# --
-# HTTP responses: ----------
-# handle name : rep-1
-# prediction  : (pid: 62636); path: /model/rep-1.pkl; data: 0.582; prediction: 1.481
-# handle name : rep-2
-# prediction  : (pid: 62637); path: /model/rep-2.pkl; data: 0.778; prediction: 1.678
-# handle name : rep-1
-# prediction  : (pid: 62634); path: /model/rep-1.pkl; data: 0.139; prediction: 0.139
-# handle name : rep-2
-# prediction  : (pid: 62635); path: /model/rep-2.pkl; data: 0.569; prediction: 1.262
+    # our request.
+    async def predict(self, data: float) -> str:
+        pred = self.model.predict(data)
+        return (f"(pid: {self.pid}); path: {self.path}; "
+                f"data: {float(data):.3f}; prediction: {pred:.3f}")
+
+    async def __call__(self, http_request: starlette.requests.Request) -> str:
+        data = float(await http_request.query_params['data'])
+        return await self.predict(data)
+
+
+@serve.deployment
+class ServeHandleDemo:
+    def __init__(self, predictor_1: Predictor, predictor_2: Predictor):
+        self.predictor_1 = predictor_1
+        self.predictor_2 = predictor_2
+
+    async def run(self):
+        for i in range(2):
```

Comment:
```suggestion
        for _ in range(2):
```

Why do we do 2 iterations here?

Reply:
Good point.  It's to demonstrate the requests getting distributed to the two replicas on each deployment, but I need to add a comment here explaining this.